src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="calendar">
+    <div class="calendar__body">
+      <div class="calendar__day calendar__day--mon calendar__day--1 calendar__day--hover"></div>
+      <div class="calendar__day calendar__day--tue calendar__day--2 calendar__day--hover"></div>
+      <div class="calendar__day calendar__day--wed calendar__day--3 calendar__day--hover"></div>
+      <div class="calendar__day calendar__day--thu calendar__day--4 calendar__day--hover"></div>
+      <div class="calendar__day calendar__day--fri calendar__day--5 calendar__day--hover"></div>
```

Comment:
```suggestion
  <body>
    <div class="calendar calendar--start-day--sun calendar--days--27">
      <div class="calendar__day"></div>
      
      <div class="calendar__day"></div>
      
      <div class="calendar__day"></div>
      
      <div class="calendar__day"></div>
      
      <div class="calendar__day"></div>
      
```

Reply:
use nth-child to interact with days

don't add modificator to each day, just for a container